startinstance/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@eformservice/startinstance",
+  "servicename": "startinstance",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ext .ts,.tsx ./",
+    "test": "cross-env HOTRELOAD=no DEBUG=bpmn-engine:123* ts-mocha -p tsconfig.json ./**/*.roxtest.ts --exit --recursive --timeout 40000 --require regenerator-runtime",
+    "copyproduction": "cross-var copyfiles package.json production && cd ./production && npm install --production && cd .. && cross-var copyfiles -u 1 production/node_modules/** $npm_package_servicename && rimraf production/",
+    "copyfiles": "cross-var copyfiles service.json dist/* $npm_package_servicename && npm run copyproduction",
+    "zip": "cross-var bestzip $npm_package_servicename.zip $npm_package_servicename",
+    "copyandzip": "cross-var npm run copyfiles && npm run zip && cross-var rimraf $npm_package_servicename",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "processhub-sdk": "https://github.com/roXtra/processhub-sdk/releases/download/v8.10.0-3/release.tgz",
```

Comment:
Please don't adjust the dependency only here. Instead, adjust the `processHubSDKVersion` in the `buildScript.js` in the root folder and afterwards run `npm run buildonly` on your machine.
All services will then be updated to link against the same sdk version. Afterwards, please commit all `package.json` and lock files.